apps/crn-server/src/app.ts

```diff
@@ -43,8 +43,18 @@ import ResearchTags, {
 } from './controllers/research-tags';
 import Teams, { TeamController } from './controllers/teams';
 import Users, { UserController } from './controllers/users';
-import AssetDataProvider from './data-providers/assets.data-provider';
-import UserDataProvider from './data-providers/users.data-provider';
+import {
+  AssetDataProvider,
+  AssetSquidexDataProvider,
+} from './data-providers/assets.data-provider';
+import {
+  GroupDataProvider,
+  GroupSquidexDataProvider,
+} from './data-providers/groups.data-provider';
+import {
+  UserDataProvider,
+  UserSquidexDataProvider,
```

Comment:
Why is there a SquidexDataProvider and a general data provider interface?

Reply:
pretty sure that's exactly what you said you could understand in the comment bellow: `I would have thought the interface would be generic whilst the implementation would be Squidex specific.`?